packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -191,6 +191,7 @@ export const CommentsItem = ({
   excerptLines,
   className,
   classes,
+  allowVoting=true,
```

Comment:
This parameter would probably be better placed inside the `treeOptions` argument which holds settings relevant to the entire comment tree. You also wouldn't need to manually pass it from `CommentsNode` into `CommentsItemMeta` and `CommentsItem` then.

I also think it'd be cleaner to have a `disableVoting` optional boolean parameter that we only set to true in the rare cases that we want it, rather than `enableVoting` with a default value of true.

Reply:
done